src/app/helpers/page-data-utils.ts

```diff
@@ -2,16 +2,24 @@ import React from 'react';
 import bookPromise from '~/models/book-titles';
```

Comment:
Changes to page-data-utils are (mostly) more easily viewed in individual commits. See "Resolve all lint issues"